jobscheduler/scheduler-component-shared/pom.xml

```diff
@@ -65,6 +69,15 @@
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-core</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This means that opencsv ends up in shared, it's already used in other parts of Sakai